Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.ps1

```diff
@@ -114,44 +133,44 @@ function Get-ProjectsToBuild(
     $modifiedFiles = @(),
     [Parameter(HelpMessage = "The maximum depth to build the dependency tree", Mandatory = $false)]
     $maxBuildDepth = 0
-) 
+)
 {
     Write-Host "Determining projects to build in $baseFolder"
-    
+
     Push-Location $baseFolder
 
     try {
-        $settings = ReadSettings -baseFolder $baseFolder -project '.' # Read AL-Go settings for the repo
-        
+        $settings = $env:Settings | ConvertFrom-Json
```

Comment:
This makes settings a PSCustomObject (earlier it was a hashtable) - is that a problem?

Reply:
I don't think it is. Any specific scenario to look for? Settings always contain the default value, right?

Reply:
HashTable:
.containsKey or .Keys + used as parameter in a number of functions in AL-Go expecting hashtables.
PSCustomObject:
.PSObject.Properties.name -eq 'x'

Reply:
You are not using any of those, but is it a problem that in some actions we work with settings as a hashtable and in other actions we work with it as PSCustomObject? Could maybe lead to misunderstandings down the road?
(we might have the same issue in other places - so OK with me)